content/news/_index.md

```diff
@@ -1,4 +1,5 @@
 ---
+layout: "news"
```

Comment:
```suggestion
```

You don't need to specify the layout here because:
1. This page is a first-level sub-directory under content, `/news`. Hugo considers every markdown file in this directory to have the [content type](https://gohugo.io/content-management/types/) "news".
2. The template for this page is located in `layouts/news/list.html`, which is [where hugo will look for a layout](https://gohugo.io/templates/lookup-order/) that is type news.

From the Hugo docs:
> Type
> Is value of type if set in front matter, else it is the name of the root section (e.g. “blog”). It will always have a value, so if not set, the value is “page”.

Reply:
Good point - fixed in 79254d3